.schemastore/config.schema.json

```diff
@@ -126,6 +126,60 @@
         "config"
       ]
     },
+    "httpRequestConfig": {
+      "type": "object",
+      "properties": {
+        "url": {
+          "title": "HTTP address of API endpoint",
+          "description": "This URL will be used to send the emails to.",
+          "examples": [
+            "https://example.com/api/v1/email"
+          ],
+          "type": "string",
+          "pattern": "^https?:\\/\\/.*"
```

Comment:
I would probably allow http in dev mode

Reply:
Right now HTTP is always allowed, as it is for other URLs. If we want to always disable HTTP when dev mode is not active, do we want to do it more generally here?

https://github.com/ory/kratos/blob/68cb201d4437f49a559239f70e6d5259aa9fe564/request/builder.go#L45-L62

Reply:
That's also fine! I overlooked the `?` instruction